package.json

```diff
@@ -24,7 +24,8 @@
     "should": "^6.0.3",
     "supertest": "^1.0.1",
     "winston": "^2.1.1",
-    "heapdump":"*"
+    "heapdump":"*",
+    "wriocommon"  : "michbil/WRIOOS-Common"
```

Comment:
can we use my repo during active development process and switch then to webrunes?